### ISSUE TITLE
Can't Select Text in Typewriter Mode

### ISSUE THREAD

**Issue by decon-structed** [olivierkes/manuskript#783]
When in typewriter mode, if I select text close to the bottom of the editor, as I move my cursor down to highlight more text, the cursor/editor jumps down to the bottom of my text, highlighting way more than I wanted. It's almost impossible to highlight the text I want when typewriter mode is on.

**Comment by siliconserf** [comment]
Yes. That's commonly observed and rather annoying. I'm pretty sure it is a flaw in a library that Manuskript includes, but doesn't directly control. The fix needs to be done in the project that supports Markdown. :-(

**Comment by TheJackiMonster** [comment]
Highlighting is mostly related to Qt rather than Markdown. I don't use the typewriter mode that much so I have to look into its source code to know if there's something which can be done to fix it.

**Comment by mxamber** [comment]
I just ran into something similar (if you select into the next line, that line is then considered the current line and centered without changing cursor position on screen, so the cursor is one line below current, repeat forever), is that what OP describes or a different issue?

I imagine a possible fix for my issue could be to temporarily disable typewriter mode while text is being selected.
